packages/core/src/services/documentTriggers/triggerEvents/runFromEvent.ts

```diff
@@ -126,20 +123,3 @@ export async function runDocumentFromTriggerEvent<
   if (!Result.isOk(runResult)) return runResult
   return Result.nil()
 }
-
-export async function enqueueRunDocumentFromTriggerEventJob({
-  workspace,
-  documentTriggerEvent,
-}: {
-  workspace: Workspace
-  documentTriggerEvent: DocumentTriggerEvent
-}): PromisedResult<undefined> {
-  const jobData: ExecuteDocumentTriggerJobData = {
-    workspaceId: workspace.id,
-    documentTriggerEventId: documentTriggerEvent.id,
-  }
-
-  await documentsQueue.add('runDocumentTriggerEventJob', jobData)
-
-  return Result.nil()
-}
```

Comment:
This was breaking the FIXME at the top because under this service is all the Mailer stuff. We just want in backoffice to enqueue the job so I moved this to another file